WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
That isn't how it works. De jure titles reflects existant structures, but if those structures are those of the ERE or associated to the Christian church, they have restrictions and requires the player to adapt, or replace these structures through dynamic evolution.